gitlab/objects.py

```diff
@@ -1353,6 +1353,23 @@ def builds(self, **kwargs):
                                      {'project_id': self.project_id},
                                      **kwargs)
 
+    def cherry_pick(self, branch, **kwargs):
+        """Cherry-pick a commit into a branch.
+
+        Args:
+            branch (str): Name of target branch.
+
+        Raises:
+            GitlabCherryPickError: If the cherry pick could not be applied.
+        """
+        url = '/projects/{0:s}/repository/commits/{1:s}/cherry_pick'.format(
+            self.project_id, self.id)
```

Comment:
For the sake of consistency could you use the old ``%()s`` format? Yes, this is nitpicking :)